src/main/java/frc/robot/commands/OuttakeAmpManual.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.constants.ArmConstants;
+import frc.robot.constants.ShooterConstants;
+import frc.robot.constants.StorageIndexConstants;
+import frc.robot.subsystems.gpm.Arm;
+import frc.robot.subsystems.gpm.Shooter;
+import frc.robot.subsystems.gpm.StorageIndex;
+
+public class OuttakeAmpManual extends SequentialCommandGroup { 
+    
+    private boolean AmpOrder = true; 
```

Comment:
This won't work. The constructor is only called once, when the command is created, so this will always only be the first step. One way to fix this is to add:
new InstantCommand(()->{AmpOrder=!AmpOrder;}),
new ConditionalCommand(
/*step 1*/,
/*step 2*/,
()->AmpOrder
)
It should also be initialized to false if you switch to this.